pluginExamples/Analysis/LocalBaseline.py

```diff
@@ -20,13 +20,13 @@ def get_configuration(self):
         )
         
         self.field(
-            name = "Peak Neighborhood",
-            initial = 3, minimum = 0, maximum = 10000, step = .2,
+            name = "Left",
+            initial = 10, minimum = 0, maximum = 10000, step = .2,
             datatype = "float", direction = "input"
-        )
-
+        )        
+        
         self.field(
-            name = "Baseline Extent",
+            name = "Right",
             initial = 10, minimum = 0, maximum = 10000, step = .2,
```

Comment:
Maybe set precision=1, since otherwise there's a frustratingly unreachable ".x0" at the end?

Reply:
EnlightenPluginFields with type="float" will always display two digits after the decimal no matter the value of step. The user can set the precision of 0.01 using the keyboard, also no matter the value of step. The step parameter only dictates how much the value changes using arrow-keys and scrollwheel.

Also, when adjusting the step param we want to be mindful that that will affect the scrollwheel speed. If we set it low then it will take a long time to move to the target value-- if we set it high then it will be easy to overshoot.

In any event, to remove the trailing 0, we would need to modify the bit of code that reads the EPF (dataclass) and generates widgets. QtSpinBox will have an option for number of digits after the decimal. I believe that is within `configure_gui_for_module` in PluginController.py